css/src/components/tag.scss

```diff
@@ -38,6 +38,9 @@ $tag-border-color-primary: $primary-background-glow-high-contrast !default;
 $tag-interactive-divider-color-primary: $primary-box-shadow !default;
 $tag-interactive-color-primary-hover: $primary-dark-hover !default;
 
+$tag-suggestion-radius: 0.5rem !default;
+$tag-suggestion-margin: 0.5rem !default;
```

Comment:
Margin/spacing outside of the element itself should just be an atomic. Adding on the component itself reduces the flexibility of the component.